hack/kfc-dev.js

```diff
@@ -0,0 +1,155 @@
+// Helper for using a KFC release candidate in Pepr
+const { exec, execSync } = require("child_process");
+const fs = require("fs");
+const p = require("path");
+
+const command = process.argv[2];
+const flag = process.argv[3];
+const path = process.argv[4];
+
+// Scaffolding
+const repoUrl = `https://github.com/defenseunicorns/kubernetes-fluent-client.git`;
+let currentDir = process.cwd();
+const moreInfo = "\n\nUse node hack/kfc-dev.js for more information.";
+
+const usage = `
+Helper for using a KFC release candidate in Pepr. 
+
+Usage:
+  node hack/kfc-dev.js [command]
+
+Available Commands:
+  build       Builds a dev image from a repo or local path
+  import      Imports Kubernetes Fluent Client source into node_modules for pepr-test-module
+
+Flags:
+  -r, --repo string string  repo and branch to clone from
+  -l, --local string        path of KFC source code
+
+Examples: 
+  # Build dev image from remote repo
```

Comment:
```suggestion
  # Build dev image from specific branch
```

This isn't actually trying to pull from `remote repos`... is it?

Reply:
yep, that is true. should be specific branch